06_iito/index.tex

```diff
@@ -1,9 +1,12 @@
 \chapter{Inter Island Trade Organisation (IITO)}
```

Comment:
let's change the title meanwhile:
"Inter-Island .."

yes, the dash is missing